packages/react-menu/etc/react-menu.api.md

```diff
@@ -17,6 +17,15 @@ import { usePopperMouseTarget } from '@fluentui/react-positioning';
 // @public
 export const Menu: React_2.FC<MenuProps>;
 
+// @public (undocumented)
+export type MenuCheckedValueChangeData = {
+    name: string;
+    checkedItems: string[];
```

Comment:
should we let users more strictly type their values ?

```suggestion
export type MenuCheckedValueChangeData<TCheckedItems = string> = {
    name: string;
    checkedItems: TCheckedItems[];
```

Reply:
it might require some more effort in the typings to make it fully work, so not a blocker